pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "whist-core"
 # remember to also update the version in __init__.py!
-version = "0.7.0"
+version = "0.7.1"
```

Comment:
why the version increase already?

Reply:
This fix is required for next steps in server so we need to release it 😉